backend/src/crd/controller/viewer/reconciler/reconciler.go

```diff
@@ -180,6 +181,11 @@ func setPodSpecForTensorboard(view *viewerV1beta1.Viewer, s *corev1.PodSpec) {
 		// when https://github.com/kubeflow/pipelines/issues/2514 is done
 		// "--bind_all",
 	}
+
+	if strings.HasPrefix(view.Spec.TensorboardSpec.TensorflowImage, `tensorflow/tensorflow:2.`) {
```

Comment:
Just curious, could you try if --bind_all flag works with 1.15?

Reply:
The flag will cause tensorflow/tensorflow:1.15.0 throw an error :"error: unrecognized arguments: --bind_all"

Reply:
Another question: if later tensorflow images start to use 3.xx, we probably still need to append bind_all flag...